.github/workflows/main.yml

```
@@ -92,3 +92,25 @@ jobs:
       # The Windows CI transforms the `c-library` symlink into a real directory, modifying the Git
       # state, so we ignore these changes with `--allow-dirty` here.
       run: scripts/verify-publish.sh --allow-dirty
+
+  itt_python_build_and_test:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+          - os: windows-latest
+    runs-on: ['${{matrix.os}}']
```

```suggestion
    runs-on: ${{ matrix.os }}
```
runs-on expects a single string value or an array of strings directly, not an array containing a single string with interpolation